RELEASE.md

```diff
@@ -171,9 +171,16 @@ Then, the Release Manager releases and announces the new version:
 
 -   [ ] 1.  If all is good give a public OK to the release
 
--   [ ] 2.  Tag the CLI, Client Side Tools, and Coverage Reporter with the version of the release being done.
-
--   [ ] 3.  Inform all stakeholders the release is finished 
+-   [ ] 2.  Tag the CLI and Coverage Reporter with the version of the release being done. **Note**: This process will be improved in [this epic](https://codacy.atlassian.net/browse/REL-51)
+    1. Go to the repository (or clone it)
+    2. Checkout the version that was validated to work with the current release
+        - `git checkout y.y.y`
+    3. Tag the commit with the current release version prefixed with `self-hosted-`
+        - `git tag self-hosted-x.x.x`
+    4. Push the new tag
```

Comment:
Codacy found an issue: [[ordered-list-marker-value] Marker should be `6`, was `4`](https://app.codacy.com/gh/codacy/chart/pullRequest?prid=6190172)